### DISCUSSION TITLE
Getting 400! error

### DISCUSSION THREAD

**Discussion by TheQueenofKings** [victornpb/undiscord#503] [2. Discussions]
So I tried inputting the IDs and Auth token manually but still got this error:
Error searching messages, API responded with status 400!
	{"code":50035,"errors":{"max_id":{"_errors":[{"code":"NUMBER_TYPE_COERCE","message":"Value \" (emailAddress)\" is not snowflake."}]}},"message":"Invalid Form Body"}
CoreException	{}

**Comment by rockettdev** [comment]
yep, they have some terrible error handling with no guide to solve these error codes.

**Comment by andy-mcpherson** [comment]
I had the same issue. For me the problem was the browser auto-filling some settings in a hidden tab. Make sure streamer mode is off so you can see all of the values and check under Messages Interval. The browser had added auto-filled my email address there and removing that solved the problem. Hope that helps
